xds/src/main/java/io/grpc/xds/XdsNameResolverProvider.java

```diff
@@ -64,12 +64,12 @@ private XdsNameResolverProvider(String scheme,
    * and bootstrap.
    */
   public static XdsNameResolverProvider createForTest(String scheme,
-                                                      @Nullable Map<String, ?> bootstrapOverride) {
```

Comment:
Revert the whitespace changes to this file. You didn't make any real changes.

Reply:
Reverted.